config/7.10.0/common.yaml

```diff
@@ -613,6 +596,7 @@ servers:
                 service: "[[.KieName]]"
                 deploymentConfig: "[[.KieName]]"
                 services.server.kie.org/kie-server-id: "[[.KieServerID]]"
+                cluster: "jgrp.k8s.[[$.ApplicationName]].[[.KieName]]"
```

Comment:
for KIE Servers only, you can get rid of [[. ApplicationName]]... so -
`"jgrp.k8s.[[.KieName]]"`

Reply:
Why not use the same pattern with all names ?

Reply:
Done